lib/arclight/traject/ead2_config.rb

```diff
@@ -267,7 +269,15 @@
   end
 
   to_field 'access_subjects_ssm' do |_record, accumulator, context|
-    accumulator.concat Array.wrap(context.output_hash['access_subjects_ssim'])
+    accumulator.concat(context.output_hash.fetch('access_subjects_ssim', []))
+  end
+
+  # Indexes the acquisition group information into the notes field
+  # Please see https://www.loc.gov/ead/tglib/elements/acqinfo.html
+  to_field 'acqinfo_ssim', extract_xpath('//xmlns:acqinfo/xmlns:note')
```

Comment:
I'm noticing that these xpaths may be searching the entire document (not just the selected component). 

I'm assuming here that for the `alphaomegaalpha.xml` fixture `acqinfo` should only be available at the top level document (not the child components)?

https://github.com/projectblacklight/arclight/blob/master/spec/fixtures/ead/nlm/alphaomegaalpha.xml#L199-L202

Maybe @anarchivist can weigh in on that expectation?

Reply:
Also, one quick question about how/if this removes the need for:
https://github.com/projectblacklight/arclight/blob/master/lib/arclight/traject/ead2_config.rb#L278